Makefile

```diff
@@ -1806,14 +1799,15 @@ rustup-install-target-toolchain: rustup-set-version
 # changelog generates PR changelog between the provided base tag and the tip of
 # the specified branch.
 #
-# usage: make -s changelog
-# usage: make -s changelog BASE_BRANCH=branch/v13 BASE_TAG=13.2.0
-# usage: BASE_BRANCH=branch/v13 BASE_TAG=13.2.0 make -s changelog
+# usage: make changelog
+# usage: make changelog BASE_BRANCH=branch/v13 BASE_TAG=13.2.0
```

Comment:
I made a little mistake here on the implementation. `BASE_TAG` needs the `v` now since it isn't added when it queries for tags. Think I accidentally deleted it during some of the refactoring.

It should be `make -s changelog BASE_BRANCH=branch/v13 BASE_TAG=v13.2.0`